stateful-ksession/pom.xml

```diff
@@ -36,6 +33,14 @@
         </license>
     </licenses>
 
+    <properties>
+        <version.org.jboss.bom.brms>6.2.0-SNAPSHOT</version.org.jboss.bom.brms>
+        <version.org.jboss.bom.eap>6.4.2.GA</version.org.jboss.bom.eap>
```

Comment:
Should we update this one  to 6.4.3.GA since CR28 is align to 6.4.3?
I am going to do ER2 base on 6.4.3 eap dependencies.